recipes/plottr/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "plottr" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7800f0ba704bfbfa0564538c790a3551142bb5e23cbe7282e69d4f9ffe0d9536
+
+build:
+  number: 0
+  skip: true  # [py27]
+  skip: true  # [py36]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxext') }}     # [linux]
+  host:
+    - xorg-libxfixes  # [linux]
+    - python
+    - pip
+  run:
+    - h5py >=2.8.0
+    - lmfit >=1.0
+    - matplotlib >=3.0.0
+    - numpy >=1.12.0
+    - python
+    - pyqtgraph >=0.10.0
+    - qtpy >=1.9.0
+  run_constrained:
+    - pyside2 >=5.12
+    - pyqt
+
+test:
+  requires:
+    - pyqt
+  imports:
+    - plottr
+
+about:
+  home: https://github.com/toolsforexperiments/plottr
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'A tool for live plotting and processing data'
+  description: |
+    A framework for inspecting data, based on flowcharts from pyqtgraph.
+    plottr's main aim is to allow the user to define custom data processing flows and plotting.
+    A particular use case is data filtering and plotting.
+  doc_url: https://plottr.readthedocs.io/en/latest/
+  dev_url: https://github.com/toolsforexperiments/plottr
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
+    - trevormorgan
```

Comment:
shall we add wolfgang, Jens, and myself as well so that more people can work on this if needed?